src/blocks/timeline/day-align-classes.js

```diff
@@ -5,12 +5,12 @@
 function DayAlignClass( attributes, index_val ) {
 	let dayAlignClass = '';
 
-	if ( 'left' == attributes.timelinAlignment ) {
+	if ( 'left' === attributes.timelinAlignment ) {
 		dayAlignClass = 'uagb-timeline__day-new uagb-timeline__day-left';
-	} else if ( 'right' == attributes.timelinAlignment ) {
+	} else if ( 'right' === attributes.timelinAlignment ) {
 		dayAlignClass = 'uagb-timeline__day-new uagb-timeline__day-right';
-	} else if ( 'center' == attributes.timelinAlignment ) {
-		if ( index_val % 2 == '0' ) {
+	} else if ( 'center' === attributes.timelinAlignment ) {
+		if ( index_val % 2 == '0' ) { // eslint-disable-line eqeqeq
```

Comment:
Can we do something like this?
```suggestion
		if ( index_val % 2 === 0 ) {
```